framework/lib/theme/components/toolbox/index.ts

```diff
@@ -2,21 +2,25 @@ import { ComponentMultiStyleConfig } from '@chakra-ui/react'
 
 export const Toolbox: ComponentMultiStyleConfig = {
   parts: [ 'container', 'header', 'body', 'footer' ],
-  baseStyle: ({ theme: { sizes: sizing } }) => ({
+  baseStyle: ({ theme: {
+    colors: color,
+    sizes: sizing,
+    borders: borderWidth,
+  } }) => ({
     container: {
       position: 'relative',
       bg: 'text.inverted',
       h: '100vh',
-      borderLeftWidth: '1px',
+      borderLeftWidth: borderWidth.xs,
       borderLeftStyle: 'solid',
-      borderLeftColor: 'gray.100',
+      borderLeftColor: color.border.default,
```

Comment:
border.default is equivalent to gray.200, not gray.100, is this change intentional?

Reply:
Yup. Part of making the movement "Figma 1:1 with NL".

Reply:
Okay I can see in the sketches that it was supposed to be border.default from the start, nvm